### ISSUE TITLE
Event gate option "[Toggle] Landing Gear" doesn't behave as a toggle 

### ISSUE THREAD

**Issue by CurioInventorium** [tsunamayo/Starship-EVO#4130]
### Short Description of the issue:

The [Toggle] Landing Gear event doesn't behave as a toggle, only as 'On' while key held down.

### Steps to reproduce the issue:

I jumped in one of my existing ships and pressed 'G' to toggle the landing gear and nothing happened.
So I went across to a work-in-progress ship and added some rudimentary piston landing gear.
After hooking up the yoke to the event gate I picked the [Toggle] Landing Gear and then wired that to a sequencer logic block to control the distance the pistons move.
The event gate output is also set to toggle.
When controlling the ship pressing 'G' to toggle the landing gear has no effect unless you hold down the 'G' key, at which point the pistons extend. And they retract when the key is released.

### Environment
- Starship EVO version: 21w33b
- Graphics Card: RTX2060

### Additional information:
I tested the behaviour of the pulse option and that worked as expected with an 'On' signal being sent for the duration specified.
Also, on my initial ship I tested the landing gear on I did find that the event gate '1' hotkey toggle option did work as expected.
So I'm guessing that the landing gear toggle option has been set to a zero-second pulse or something.

I'm also wondering why we have duplicate options for toggle and pulse listed on the event gate, when there's a control box we can click on to choose if the signal is Pulse or Toggle.

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by tsunamayo** [comment]
@CurioInventorium Yes I might have to change the [Toggle] denomination as it is getting confusing for hotkey. In toggle mode the key will be On only when you press it. So if you did not feed that into a true switch, it will simply desactivate.
I could also add a make switch option, so it would behave like a builtin switch.

**Comment by TIKIRobo** [comment]
@tsunamayo  please do add a switch option, most of my landing gears use that because i dont like hearing the little click of the switch on some of my smaller fighter sized ships

**Comment by CurioInventorium** [comment]
@tsunamayo I was always under the impression that the 'toggle' mechanic, as previously used, acted as a switch to save having to add one.
Maybe change "Toggle" to "Hold for On" or something. Because I'm pretty sure "Toggle" always meant "Press to change".
At the moment there is inconsistent behaviour for hotkey 1 and landing gear. As mentioned above, hotkey 1 acts like a switch while landing gear is On when key is held down.

**Comment by Jorpis-Jorah** [comment]
Yeah this is confusing for me too, I had tried to enable landing gear but it forced me to hold down the button the entire time. :\

**Comment by CurioInventorium** [comment]
I think there'll be a lot of broken landing gears in ships if this isn't migrated correctly.

**Comment by ExodistSKY1** [comment]
I am a bit late tot he party but I would also like a switch option within the Event gate as to help keep its simplicity
